deployment.rb

```diff
@@ -128,9 +128,9 @@ def load_configuration()
     config['pegasus_reporting_db_reader'] ||= config['reporting_db_reader'] + config['pegasus_db_name']
     config['pegasus_reporting_db_writer'] ||= config['reporting_db_writer'] + config['pegasus_db_name']
 
-    # Set AWS SDK environment variables from provided config.
-    ENV['AWS_ACCESS_KEY_ID'] ||= config['aws_access_key'] || config['s3_access_key_id']
-    ENV['AWS_SECRET_ACCESS_KEY'] ||= config['aws_secret_key'] || config['s3_secret_access_key']
+    # Set AWS SDK environment variables from provided config and standardize on aws_* attributres
```

Comment:
I did this here so that I wouldn't have to repeat the aws_access_key || s3_access_key_id thing in the cloud_formation.rb code